wsd/FileServer.cpp

```diff
@@ -494,24 +490,20 @@ void FileServerRequestHandler::sendError(int errorCode, const Poco::Net::HTTPReq
                                          const std::string& shortMessage, const std::string& longMessage,
                                          const std::string& extraHeader)
 {
-    Poco::URI requestUri(request.getURI());
-    const std::string& path = requestUri.getPath();
-    std::ostringstream oss;
-    oss << "HTTP/1.1 " << errorCode << "\r\n"
-        "Content-Type: text/html charset=UTF-8\r\n"
-        "Date: " << Util::getHttpTimeNow() << "\r\n"
-        "User-Agent: " << WOPI_AGENT_STRING << "\r\n"
-        << extraHeader
-        << "\r\n";
+    std::string body = "";
```

Comment:
std::string body; 

The default construct is already good enough, no advantage in passing an empty string.

Reply:
resolved in 3d9ee4292bdcb88ca66b2ca11826eccb8ee488fe